Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submissions-grid/SubmissionsGrid.tsx

```diff
@@ -76,7 +77,11 @@ const SubmissionsGrid = ({
                     <thead>
                         <tr className={headerClassName}>
                             <td>ID</td>
-                            <td>Task</td>
+                            {
+                                options.showTaskDetails
```

Comment:
here I think using && operator for the rendering check would add to readability, instead of using ternary operator